Snippets/Snippets_6/Transactions.cs

```diff
@@ -32,17 +50,17 @@ public void TransportTransactionsWithScope()
         {
             #region TransactionsWrapHandlersExecutionInATransactionScope
             BusConfiguration busConfiguration = new BusConfiguration();
-            busConfiguration.Transactions()
-                .WrapHandlersExecutionInATransactionScope();
+            busConfiguration.UnitOfWork()
+                .WrapHandlersInATransactionScope();
             #endregion
         }
 
-        public void CustomTransactionTimeout()
+        public void UnitOfWorkWrapHandlersInATransactionScope()
         {
-            #region CustomTransactionTimeout
+            #region UnitOfWorkWrapHandlersInATransactionScope
```

Comment:
You need to rename the corresponding region in the v5, v4 and v3 snippets as well (if you want them to show in the same place)